deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: push.aerogear.org/v1alpha1
+kind: UnifiedPushServer
+metadata:
+  name: example-ups-with-backups
+spec:
+  backups:
```

Comment:
When thinking about this originally, I was thinking about something more flexible like this: https://gist.github.com/grdryn/034d654a32af7dadd79ba1c47c17ee60

Here in this PR though, I've just copied the integreatly way, as done in the keycloak operator, for consistency. Unfortunately that ties us to some things that are not very transparent (and required me reading most of the code in [integr8ly/backup-container-image](https://github.com/integr8ly/backup-container-image) to see what was needed. Hence the elaborate comments in here.